File: src/test/java/com/kevin/StudentTest.java
package com.kevin;

import com.kevin.kotlin.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    void highestTest() {
        Student student = new Student("Kevin", 77, 11);
        Assertions.assertEquals(77, student.highest());
    }

    @Test
    void averageTest() {
        Student student = new Student("Kevin", 77, 11);
        Assertions.assertEquals((77 + 11) / 2, student.getAverage());
    }
}

File: src/test/kotlin/StudentTest.kt
import com.kevin.kotlin.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun highestTest() {
        val student = Student("Kevin", 77, 88)
        Assertions.assertEquals(77, student.english)
    }

    @Test
    fun averageTest() {
        val student = Student("Kevin", 77, 88)
        Assertions.assertEquals((77 + 88) / 2, student.getAverage())
    }

    @Test
    fun gradingTest() {
        val student = Student("Kevin", 77, 88)
        Assertions.assertEquals('B', student.grading())
    }
}